tools/packages/taipy-gui/setup.requirements.txt

```diff
@@ -1,8 +1,8 @@
 charset-normalizer>=3.3.2,<=3.3.2
-flask>=3.0.0,<=3.0.3
+flask>=3.1.0,<3.2
 flask-cors>=5.0.0,<5.1
-flask-socketio>=5.3.6,<=5.3.6
-gevent>=23.7.0,<=24.2.1
```

Comment:
should pyarrow be here ?

Reply:
it is declared in the setup.py file on the same folder